lib/pathutils.c

```diff
@@ -26,6 +26,8 @@
 #include "pathutils.h"
 #include <sys/stat.h>
 #include <string.h>
+#include <stdio.h>
```

Comment:
Maybe `msg_error` could be used instead of `fprintf`.

Reply:
Thanks, fixed.